plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -51,6 +53,19 @@ public IcebergConfig setCatalogType(CatalogType catalogType)
         return this;
     }
 
+    @Config("iceberg.catalog.warehouse")
```

Comment:
not sure about the name. something like "default storage location"?


cc @losipiuk @alexjo2144 @phd3

Reply:
This is to match https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/CatalogProperties.java#L31. Please let me know if other name is preferred, I can update to that.

Reply:
For schemas and tables we use "location". So probably we should use it also here. `iceberg.default-schema-location`?